sensugrid.py

```diff
@@ -17,17 +21,18 @@
     get_stashes,
     get_filter_data,
     get_clients,
-    get_events
+    get_events,
+    filter_data,
 )
 from gridconfig import DevConfig
 
-from multiprocessing.dummy import Pool as ThreadPool
-# https://stackoverflow.com/questions/2846653/how-to-use-threading-in-python
-
 import json
 import logging
 import logging.config
 
+from diskcache import Cache
+cache = Cache('/tmp/sensugrid')
```

Comment:
this is probably better off somewhere in /var/tmp/ as /tmp/ gets cleaned up periodically - that might cause some confusion to the app when the file is suddenly gone